src/python/WMCore/WMSpec/Steps/Executors/LogCollect.py

```diff
@@ -62,38 +71,39 @@ def execute(self, emulator = None):
-        pnn       = overrides.get('phedex-node', "T2_CH_CERN")
-        lfnPrefix = overrides.get('lfn-prefix', "srm://srm-cms.cern.ch:8443/srm/managerv2?SFN=/castor/cern.ch/cms")
-        lfnBase   = overrides.get('lfnBase',   "/store/user/amaltaro")
-        userLogs  = overrides.get('userLogs',  False)
-
-        stageOutParams = {"command": "srmv2-lcg",
-                          "se-name": seName,  "phedex-node": pnn, "lfn-prefix": lfnPrefix}
-
-        # Set EOS stage out params
-        seEOSName    = "srm-eoscms.cern.ch"
-        lfnEOSPrefix = "srm://srm-eoscms.cern.ch:8443/srm/v2/server?SFN=/eos/cms"
-        
-        stageEOSOutParams = {"command": "srmv2-lcg",
-                             "se-name": seEOSName, "phedex-node": pnn, "lfn-prefix": lfnEOSPrefix}    
-
-        # Okay, we need a stageOut Manager
+        waitTime = overrides.get('waitTime', 3600 + (self.step.retryDelay * self.step.retryCount))
+
+        # Pull out StageOutMgr Overrides
+        # switch between old stageOut behavior and new, fancy stage out behavior
+        useNewStageOutCode = False
+        if 'newStageOut' in overrides and overrides.get('newStageOut'):
+            useNewStageOutCode = True
+
+        # hardcode CERN Castor T0_CH_CERN_MSS stageout parameters
+        castorStageOutParams = {}
+        castorStageOutParams['command'] = overrides.get('command', "srmv2-lcg")
+        castorStageOutParams['option'] = overrides.get('option', "")
+        castorStageOutParams['se-name'] = overrides.get('se-name', "srm-cms.cern.ch")
+        castorStageOutParams['phedex-node'] = overrides.get('phedex-node', "T2_CH_CERN")
+        castorStageOutParams['lfn-prefix'] = overrides.get('lfn-prefix', "srm://srm-cms.cern.ch:8443/srm/managerv2?SFN=/castor/cern.ch/cms")
+
+        # hardcode CERN EOS T2_CH_CERN stageout parameters
+        eosStageOutParams = {}
+        eosStageOutParams['command'] = overrides.get('command', "srmv2-lcg")
+        eosStageOutParams['option'] = overrides.get('option', "")
+        eosStageOutParams['se-name'] = overrides.get('se-name', "srm-eoscms.cern.ch")
+        eosStageOutParams['phedex-node'] = overrides.get('phedex-node', "T2_CH_CERN")
+        eosStageOutParams['lfn-prefix'] = overrides.get('lfn-prefix', "srm://srm-eoscms.cern.ch:8443/srm/v2/server?SFN=/eos/cms")
+
+        # are we using the new stageout method ?
         useNewStageOutCode = False
         if getattr(self.step, 'newStageout', False) or \
             ('newStageOut' in overrides and overrides.get('newStageOut')):
             useNewStageOutCode = True
 
         try:
-            if not useNewStageOutCode:
-                # old style
-                stageOutMgr = StageOutMgr(**stageOutParams)
-                stageOutEOSMgr = StageOutMgr(**stageEOSOutParams)
-                stageInMgr = StageInMgr()
-                deleteMgr = DeleteMgr()
-            else:
-                # new style (is even working ???)
+            if useNewStageOutCode:
```

Comment:
The best part of this code (and actually the current master code) is that there is *no* difference at all when we use the new stage out code or not... at least not when we instantiate these classes, with exactly the same arguments.

Dirk, please remove the `if/else` and the dup lines